lua/telescope/builtin/__internal.lua

```diff
@@ -712,7 +712,17 @@ internal.help_tags = function(opts)
   end
 
   local help_files = {}
-  local all_files = vim.api.nvim_get_runtime_file("doc/*", true)
+
+  -- get the current vim runtime path
```

Comment:
comment is not really needed

Reply:
Removed unnecessary comment in latest commit revision